ExpenseTracker/ExpenceTracker.Domain/Interfaces/IRepositoryBase.cs

```diff
@@ -4,10 +4,10 @@ namespace ExpenseTracker.Domain.Interfaces
 {
     public interface IRepositoryBase<TEntity> where TEntity : EntityBase
     {
-        List<TEntity> GetAll();
-        TEntity GetById(int id);
+        List<TEntity> GetAll(Guid userId);
+        TEntity GetById(int id, Guid userId);
         TEntity Create(TEntity entity);
         void Update(TEntity entity);
```

Comment:
when creating & updating, can the userId of the Guid type come as a parameter to this method, because how does the database determine which user will update or create  a transfer or category?

Reply:
I added the userId to the entity base and it is determined by it